meta-resin-common/recipes-support/resin-expand/resin-filesystem-expand/resin-filesystem-expand

```diff
@@ -5,7 +5,7 @@ set -e
 DATA_PARTITION=/dev/disk/by-label/resin-data
 
 echo -n "Expand ext4 partition on $DATA_PARTITION... "
-e2fsck -y -f $DATA_PARTITION
+e2fsck -y $DATA_PARTITION
```

Comment:
The existence of the -f flag is just impacting the first boot after provisioning in most cases, right?
Subsequent boots are not affected by this flag being there or not I guess?